### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by Demydoff** [abrechts/Phoenix-ELN#12] [General]
Thank you for your software (I can't thank you enough). I would like to know if project schematics will be added in a general form and the ability to edit the solvent and substance database? These small features would be quite useful. Also, improvements to the substance search in the database are desired. Currently, the search engine does not find words in the middle of a word (example: you cannot find a position named [MTBE (Methyl tert-butyl ether)] in the search if you enter the word Methyl). Will there be an option to merge several projects into a larger project? Or will a project folder be added? It's not crucial, but it would be convenient at times.

**Comment by abrechts** [comment]
Thanks for your inquiry. Based on your questions, I assume that you were using _Espresso ELN_ in the past. Since there may be many others having similar questions, I've added the announcement _How does Phoenix ELN relate to legacy Espresso ELN_ #13, which already might answer some of your questions. To answer some possibly remaining ones:

- Just tick the "Subtext" checkbox to the right of a material name box to initiate the in-text searches you were asking for :-).

- Its not sure yet if and in which form project summary schemes will be implemented in the future. There's currently no priority for this functionality.

**Comment by Demydoff** [comment]
Thank you for your work and product.
Do you plan to improve the calculation of yields for reactions in which the racemate is separated into R and S isomers? It is inconvenient to calculate now.

**Reply by abrechts** [reply]
I'm not sure to full understand your question. Anyway, the situation is as follows: In case of a racemate resolution you best draw your reaction sketch with the racemic structure to the left of the reaction arrow. If your desired product enantiomer is R, then first draw  the structure of the R-enantiomer to the right of the reaction arrow (i.e. as the reference product), followed by the structure of the S-enantiomer, otherwise vice versa). The maximum resolution yield can be no more than 50%, as given by the reaction stoichiometry. Phoenix ELN should calculate this correctly, if you subsequently enter the amount of your reference product (i.e. the R or S-enantiomer) within the protocol. Hopefully this helps.

**Comment by Demydoff** [comment]
![image](https://github.com/user-attachments/assets/eb6a6d7d-8f86-4ed3-aca7-717ffdcfca5d)
Hello, are you planning to add an implementation for displaying a mixture of cis/trans isomers to your editor?

**Comment by abrechts** [comment]
You added your question to a closed question about a different topic. Please always create a new discussion per new question: Just go to the Discussions section, click the 'New discussion' button, enter a meaningful title summarizing the content of your question (e.g. 'How to draw cis/trans isomers?') and add (copy) the content of your original question to it. Users will be able to find your question and subsequent discussions immediately afterwards, and it also can be closed individually when done. I'll be happy to discuss your question afterwards. Thanks!
